### ISSUE TITLE
Mute command for Spam/Annoying

### ISSUE THREAD

**Issue by Kozlov98** [haxball/haxball-issues#29]
A  /mute command to silence annoying or spammers. it would will help many official competitions

Thx for read - Kozlov

**Comment by erenseymen** [comment]
Just ban them :) Kidding. I think spectator mode (https://github.com/haxball/haxball-issues/issues/45) would be more convenient for official competitions.

**Comment by Kozlov98** [comment]
Nope, the mute command need to be placed. In official matches are better mute a Player than ban her (in camp)
